rs/pocket_ic_server/BUILD.bazel

```diff
@@ -211,6 +211,12 @@ rust_test(
         "requires-network",
         "test_macos",
     ],
+    # We temporarily disable the test on x86_64-darwin because the HTTP server of the pocket-ic-server on that platform
```

Comment:
it's the reqwest library (used by the `pocket-ic` crate) that returns an unexpected 400

Reply:
Yep. I'll document that it's caused by hyper.